mock-web-services/esri/__init__.py

```diff
@@ -0,0 +1,32 @@
+from webservice import WebService
+
+
+class EsriMockWebService(WebService):
+
+    def __init__(self, app):
+        WebService.__init__(self, app)
+        self.app = app
+        self.file_root = "../server/test/resources/esri/"
```

Comment:
**suggestion (blocking):** please mention in the doc string and also maybe the README that this loads/shares its data fixtures from the server junit test subdir